src/main/kotlin/com/autonomousapps/subplugin/ProjectPlugin.kt

```diff
@@ -531,9 +535,13 @@ internal class ProjectPlugin(private val project: Project) {
     val computeDominatorTask = tasks.register<ComputeDominatorTreeTask>("computeDominatorTree$taskNameSuffix") {
       projectPath.set(thisProjectPath)
       physicalArtifacts.set(artifactsReportTask.flatMap { it.output })
+      runtimePhysicalArtifacts.set(artifactsReportTask.flatMap { it.outputRuntime })
       graphView.set(graphViewTask.flatMap { it.output })
-      outputTxt.set(outputPaths.dominatorConsolePath)
-      outputDot.set(outputPaths.dominatorGraphPath)
+      runtimeGraphView.set(graphViewTask.flatMap { it.outputRuntime })
+      outputTxt.set(outputPaths.compileDominatorConsolePath)
+      runtimeOutputTxt.set(outputPaths.runtimeDominatorConsolePath)
+      outputDot.set(outputPaths.compileDominatorGraphPath)
+      runtimeOutputDot.set(outputPaths.runtimeDominatorGraphPath)
     }
 
     tasks.register<PrintDominatorTreeTask>("printDominatorTree$taskNameSuffix") {
```

Comment:
I didn't modify this one. Figured we'd be relying on files written to by the compute task, so including runtime sizes in that one is enough, but happy to add runtime output to this one too.